sections/transactions.js

```diff
@@ -100,7 +100,45 @@ export default [
       "Throwing an error directly from the transaction handler function automatically rolls back the transaction, same as returning a rejected promise.",
       "Notice that if a promise is not returned within the handler, it is up to you to ensure `trx.commit`, or `trx.rollback` are called, otherwise the transaction connection will hang.",
       "Calling `trx.rollback` will return a rejected Promise. If you don't pass any argument to `trx.rollback`, a generic `Error` object will be created and passed in to ensure the Promise always rejects with something.",
+    ]
+  },
+  {
+    type: "heading",
+    size: "md",
+    content: "Save Points:",
+    href: "Save-Points"
+  },
+  {
+    type: "text",
+    content: [
+      "By using save points you can roll back the transaction to an earlier state and retry statements that caused an error. Save points are like transactions within transactions, which is how you create them.",
```

Comment:
Here instead of explaining what savepoints are (out of scope for knex) we should just claim that:

Knex implements savepoints kind of functionality for allowing to rollback transactions only partly with nested transaction and then just explain how nested transactions work. In that way knex will not claim that it uses save points to implement that functionality thus leaves it open that other databases might implment them in a different way. In header it is ok to have "savepoints" mentioned, so that people may find this easier.